src/solver/ts-generator/thermal.cpp

```diff
@@ -90,9 +90,8 @@ class GeneratorTempData final
                                const T& duration);
 
 private:
-    uint nbHoursPerYear;
-
-    const uint daysPerYear;
+    const uint nbHoursPerYear = 8760;
+    const uint daysPerYear = 365;
```

Comment:
Use a specific **constant.h** header ?

Reply:
Why not ? I have currently no idea, but we'll think about it

Reply:
I think we should create a ticket for this, and not handle this in this PR, because it's a subject on its own.